integration-mc1-15/src/main/kotlin/org/eln2/items/OreChunks.kt

```diff
@@ -0,0 +1,19 @@
+package org.eln2.items
+
+import net.minecraft.client.util.ITooltipFlag
+import net.minecraft.item.Item
+import net.minecraft.item.ItemStack
+import net.minecraft.util.text.ITextComponent
+import net.minecraft.util.text.TranslationTextComponent
+import net.minecraft.world.World
+
+/**
+ * It's a chunk of ore! This is what you get when you break the ore block.
+ * Minecraft handing you a block that you can place makes ~0 sense.
+ */
+class OreChunks: Item(Properties().maxStackSize(64)) {
```

Comment:
Isn't 64 already the default?

Reply:
It may be, but I didn't see any harm in specifying. Later though, I may base this around storing NBT and having mass to reduce the number of items you store.